src/parser.rs

```diff
@@ -599,18 +598,19 @@ impl Library {
                                     "name" => field_name = attr.value.clone(),
                                     _ => {},
                                 }
+                                break;
```

Comment:
I'm not sure how many cases will have more than one field with a "name", so break in the case of finding the first one. The attributes at that stage look like

```
[OwnedAttribute { name: OwnedName { local_name: "name", namespace: None, prefix: None }, value: "u" }, OwnedAttribute { name: OwnedName { local_name: "type", namespace: Some("http://www.gtk.org/introspection/c/1.0"), prefix: Some("c") }, value: "u" }]
```